models/repo.go

```diff
@@ -149,9 +149,9 @@ const (
 
 // Repository represents a git repository.
 type Repository struct {
-	ID                  int64                  `xorm:"pk autoincr"`
-	OwnerID             int64                  `xorm:"UNIQUE(s) index"`
-	OwnerName           string                 `xorm:"-"`
+	ID                  int64 `xorm:"pk autoincr"`
+	OwnerID             int64 `xorm:"UNIQUE(s) index"`
+	OwnerName           string
```

Comment:
~~Perhaps index this column, so input like "xorm/core" can be queried just with the `repository` table?~~

I see it's `Name` and not `LowerName`. Boomer. 😔

Reply:
This column shouldn't be index, because we have `owner_id`. This column is a duplicate column only for easy repopath.